cl/recap/tests.py

```diff
@@ -279,10 +278,10 @@ def test_debug_does_not_create_rd(self, mock):
             pacer_doc_id='asdf',
             document_number='1',
             filepath_local=self.pdf,
-            upload_type=PDF,
+            upload_type=UPLOAD_TYPE.PDF,
             debug=True,
         )
-        _ = process_recap_pdf(pq.pk)
+        process_recap_pdf(pq.pk)
```

Comment:
This is probably a python convention, but doing the `_ =` bit says:

> I know this returns something and I don't care about it.

It's especially useful when something returns a tuple because you can do:

    count, _ = get_objects_and_count()

Which says, I care about the count, but forget about the actual objects. Instead of:

    count, objects = get_objects_and_count()

Which then makes you wonder if `objects` is ever going to be used (it's not).

Anyway, I think this would be better acknowledging that it has a result, but that we're tossing it, like it was prior.

Reply:
I feel like this is originally a _perl_ convention (`$_`) but yes. But flake8 still isn't happy:
`recap/tests.py:285:9: F841 local variable '_' is assigned to but never used`

so do you want a `#noqa: F841` or do you the assignment removed?

Reply:
Bleh. You can leave your change. Maybe someday Flake8 will ignore this.